client/components/repository/Revisions/EntityRevisions.vue

```diff
@@ -72,7 +73,7 @@ export default {
     rollback(revision) {
       this.$set(revision, 'loading', true);
       const entity = { ...revision.state, paranoid: false };
-      axios.patch(`/repositories/${this.repositoryId}/tes/${entity.id}`, entity)
+      return contentElementApi.patch(entity, entity)
```

Comment:
entity, entity looks a bit weird 🤔if so, we can just pass entity as single arg and tweak the api....